deploy/helm/minikube-esri-dev.yml

```diff
@@ -31,13 +31,14 @@ registry-api:
   skipAuthorization: false
   skipOpa: false
   recordPolicyId: "object.registry.record.esri_owner_groups"
-  
+
 correspondence-api:
   smtpPort: 2525
   smtpHostname: "smtp.mailgun.org"
-  
+
 web-server:
   fallbackUrl: "https://data.gov.au"
+  showNotificationBanner: false
```

Comment:
If you set it to `false` in magda-web/values.yml it'll default to false and you won't have to set it in all of these deployment specific yamls :)

Reply:
Done